### PR TITLE
Update and rename shorthand guide

### PR DESCRIPTION
**Pull request by alcuadrado**

### CODE REVIEW

File: docs/src/content/hardhat-runner/docs/guides/command-line-completion.md [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+This package, `hardhat-shorthand`, installs a globally accessible binary called `hh` that runs your locally installed `hardhat`.
 
 ## Installation
 
 To use the Hardhat shorthand you need to install it **globally**:
 
 ```
-npm i -g hardhat-shorthand
+npm install --global hardhat-shorthand
 ```
```
Comment (fvictorio) [inline docs/src/content/hardhat-runner/docs/guides/command-line-completion.md:13]: Should we use tabs here? It doesn't feel as necessary as in other places, but maybe?

**Review by fvictorio — APPROVED** [review]

File: docs/src/content/hardhat-runner/docs/guides/command-line-completion.md (hunk 1 above)
Comment (alcuadrado) [reply docs/src/content/hardhat-runner/docs/guides/command-line-completion.md:13]: I was a bit confused about this. As any of them would work. Maybe it's simpler to recommend a single option. Less to debug if people follow it and have problems.

Comment (alcuadrado) [reply docs/src/content/hardhat-runner/docs/guides/command-line-completion.md:13]: Nah, let's be consistent.

Comment (alcuadrado) [reply docs/src/content/hardhat-runner/docs/guides/command-line-completion.md:13]: Nop, I changed my mind. This works differently in yarn 1 and 2. Let's stick to npm.

**Pull request merged**